sonar-plugin-api/src/main/java/org/sonar/api/config/PropertyFieldDefinition.java

```diff
@@ -124,6 +126,10 @@ public Builder description(String description) {
       return this;
     }
 
+    /**
+     * @deprecated since 6.1, as it was only used for UI.
+     */
+    @Deprecated
     public Builder indicativeSize(int indicativeSize) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)